packages/common/src/utils/challenges.ts

```diff
@@ -259,3 +260,15 @@ export const isCooldownChallengeClaimable = (
     dayjs.utc().diff(dayjs.utc(challenge.created_at), 'day') >= 7
   )
 }
+/* Filter for only claimable challenges */
+export const getClaimableChallengeSpecifiers = (
+  specifiers: SpecifierWithAmount[],
+  undisbursedUserChallenges: UndisbursedUserChallenge[]
+) => {
+  return specifiers.filter((s) => {
+    const challenge = undisbursedUserChallenges.filter(
+      (c) => c.specifier === s.specifier
+    )[0] // specifiers are unique
```

Comment:
nit: I know this was existing code, but...
Are we certain this won't end up an empty array? `isCooldownChallengeClaimable` doesn't appear to expect or handle `undefined`.

Reply:
good call, will follow up